models/attrVoiceSampleModel.js

```diff
@@ -1,23 +1,25 @@
 const db = require('../data/dbConfig.js');
+const attribute = require('./attributesModel.js');
 
-const findAll = (id) => {
-  return db('attributes_voice_samples')
-    .where({sample_id: id})
-    .select("id");
+const findAll = async (id) => {
+  const associations = await db('attributes_voice_samples')
+    .where({voice_sample_id: id})
+  return Promise.all(associations.map(async assoc => {
+    return attribute.findById(assoc.attribute_id);
+  }))
 }
```

Comment:
f-a-n-t-a-s-t-i-c. Promise.all allowing the map to return all promises before continuing